.github/workflows/release.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         run: |
           set -euo pipefail
           gh -R "$GITHUB_REPOSITORY" release download "$GITHUB_REF_NAME" -p "*.tar.gz"
-          gh -R "$GITHUB_REPOSITORY" release download "$GITHUB_REF_NAME" -p "attestation.intoto.jsonl"
+          gh -R "$GITHUB_REPOSITORY" release download "$GITHUB_REF_NAME" -p "multiple.intoto.jsonl"
```

Comment:
if you want to preserve the old name i believe there's an input `provenance-name` as well
https://github.com/slsa-framework/slsa-github-generator/tree/77395259ff7f705e2578f4552b79d90c06f49554/internal/builders/generic#workflow-inputs

Reply:
An alternative to that would be to use the `provenance-name` output which will always be the name of the provenance artifact regardless of the input.
https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/builders/generic/README.md#workflow-outputs

Reply:
I don't have a strong preference for the filename either way, so long as verification works. Thanks for making this change!